website/content/en/docs/concepts/nodeclasses.md

```diff
@@ -160,7 +160,7 @@ status:
   # Generated instance profile name from "role"
   instanceProfile: "${CLUSTER_NAME}-0123456778901234567789"
 ```
-Refer to the [NodePool docs]({{<ref "./nodepools" >}}) for settings applicable to all providers. To explore various `EC2NodeClass` configurations, refer to the examples provided [in the Karpenter Github repository](https://github.com/aws/karpenter/blob/main/examples/v1beta1/).
+Refer to the [NodePool docs]({{<ref "./nodepools" >}}) for settings applicable to all providers. To explore various `EC2NodeClass` configurations, refer to the examples provided [in the Karpenter Github repository](https://github.com/aws/karpenter/blob/main/examples/v1/).
```

Comment:
Last comment, this should point to v0.37 
```suggestion
Refer to the [NodePool docs]({{<ref "./nodepools" >}}) for settings applicable to all providers. To explore various `EC2NodeClass` configurations, refer to the examples provided [in the Karpenter Github repository](https://github.com/aws/karpenter/blob/v0.37.0/examples/v1beta1/).
```

Reply:
It's in the "latest" section, seemingly not related to any revision, so should it?

Reply:
Yeah, the docs page point to the latests released version of Karpenter, so in this case it should be v0.37.0

Reply:
Gotcha, so I'll fix all "latest" references of this PR to 0.37.0.

Reply:
"preview" as well, or only "docs"?

Reply:
Only docs